hydrus/core/HydrusKritaHandling.py

```diff
@@ -28,45 +32,50 @@ def ExtractZippedImageToPath( path_to_zip, temp_path_file ):
         
         raise HydrusExceptions.NoThumbnailFileException( f'This krita file had no {KRITA_FILE_MERGED} or {KRITA_FILE_THUMB}!' )
         
+
+def MergedPILImageFromKRA(path):
+
+    try:
+
+        file_obj = io.BytesIO( HydrusArchiveHandling.ReadSingleFileFromZip( path, KRITA_FILE_MERGED ))
+
+        return PILImage.open( file_obj )
```

Comment:
Also you should probably use this here:
```suggestion
        return HydrusImageHandling.GeneratePILImage( file_obj )
```